etc/links.profile

```diff
@@ -22,30 +20,40 @@ include disable-xdg.inc
 mkdir ${HOME}/.links
 whitelist ${HOME}/.links
 whitelist ${DOWNLOADS}
-include whitelist-common.inc
-include whitelist-var-common.inc
 
 caps.drop all
 ipc-namespace
+# uncomment machine-id if you want to allow access only to user-configured
+# associated music player
 machine-id
 netfilter
+# uncomment no3d if you want to allow access only to user-configured
+# associated video player
 no3d
 nodvd
 nogroups
 nonewprivs
 noroot
+# uncomment nosound if you want to allow access only to user-configured
+# associated music player
 nosound
 notv
 nou2f
+# uncomment novideo if you want to allow access only to user-configured
+# associated video player
```

Comment:
`uncomment novideo (or put 'ignore novideo' in your links.local) if you want…`

Reply:
Didn't work media player with `novideo` because for example dragon has `novideo` is his profile?

Reply:
@jose1711 novideo blocks webcams mostly. It shouldn't affect playing videos.